pkg/catalog/types.go

```diff
@@ -20,7 +20,8 @@ import (
 )
 
 const (
-	Row_ID = "__rowid"
+	Row_ID           = "__mo_rowid"
+	PrefixPriColName = "__mo_cpkey_"
```

Comment:
what is cpkey?

Reply:
this is composite primary key, I just move the position and add a prefix